aspnetcore/migration/1x-to-2x/samples/AspNetCoreDotNetFx2.0App/AspNetCoreDotNetFx2.0App/Program.cs

```diff
@@ -12,6 +12,7 @@ namespace AspNetCoreDotNetFx2._0App
 {
     public class Program
     {
+        #region snippet_ProgramMainConfigProviders
```

Comment:
Regions are the devil, don't use them 😄

Reply:
I agree that they're the devil; however, these regions are used specifically to make importing of code snippets into the docs easier to maintain. Indicating line numbers has proven very brittle, especially as code samples change.